addon/templates/components/-ember-popper-legacy.hbs

```diff
@@ -0,0 +1,10 @@
+{{yield (hash
+  popper=(hash
```

Comment:
yeah, definitely agree that it would be ideal to drop these. We can even drop them now if you're ok with the breaking changes (there are others like the class one) and only have them for the 1.11 version